src/runtime/qurt_hvx.cpp

```diff
@@ -2,6 +2,7 @@
 #include "HalideRuntimeQurt.h"
 #include "printer.h"
 #include "mini_qurt.h"
+#include "hexagon_remote/log.h"
```

Comment:
I was adding the debug target feature...and passing user_context (which I have since removed, not in the current code) to the prefetch routine and didn't see any messages.  As a test I was also unable to trigger the debug messages in the other routines in this file either.  It might have been operator error, but the mechanism didn't work for me.